mitxpro/utils_test.py

```diff
@@ -62,3 +63,13 @@ def test_get_field_names():
         "created_on",
         "updated_on",
     }
+
+
+def test_partition():
```

Comment:
Can you add another test or parameterize this to test the default `bool` predictate?